optaplanner-persistence/optaplanner-persistence-jsonb/src/test/java/org/optaplanner/persistence/jsonb/api/score/buildin/bendable/BendableScoreJsonbAdapterTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jsonb.api.score.buildin.bendable;
+
+import javax.json.bind.annotation.JsonbTypeAdapter;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
+import org.optaplanner.persistence.jsonb.api.score.AbstractScoreJsonbAdapterTest;
+
+public class BendableScoreJsonbAdapterTest extends AbstractScoreJsonbAdapterTest {
+
+    @Test
+    public void serializeAndDeserialize() {
+        assertSerializeAndDeserialize(null, new TestBendableScoreWrapper(null));
```

Comment:
Not sure why does this method have the `score` argument, when it also has the `wrapper` which can return the score?

Reply:
The `score` arg is used as a reference to assert that the `Wrapper` is serialized, then deserialized to contain the same `score` result. This approach is also used in the other modules, but we could discuss changing them all for a future PR.